README.md

```diff
@@ -469,6 +472,48 @@ import { FormControl, NgValidatorsErrors } from '@ngneat/reactive-forms';
 
 const control = new FormControl<string, NgValidatorsErrors>();
 ```
+## Control Operators
+
+Each `valueChanges` or `values$` takes an operator `diff()`, which emits only changed parts of form:
+
+```ts
+import { FormControl } from '@ngneat/reactive-forms';
+
+const control = new FormControl<string>('');
+control.valueChanges
+  .pipe(diff())
+  .subscribe(value => {
+    // value is emited only if it has been changed
+  });
+```
+
+Use it with a `FormArray`:
```

Comment:
We can show only the `FormGroup` example, which is the most common. Moreover, add some more keys to the form, please.

Reply:
Sorry that it was so much time.

Fields added, I'm not sure how to show what it actually is doing.
Maybe I should write example with input and output.

Reply:
> We can show only the FormGroup example, which is the most common